server/scripts/emails.ts

```diff
@@ -173,7 +173,7 @@ export const sendImmediateNotificationEmail = async (subscription, emailObject)
   emailObject.bcc = 'raymond+bcc@commonwealth.im';
 
   try {
-    console.log('sending immediate notification email');
+    console.log(`sending immediate notification email to ${emailObject.to}`);
```

Comment:
unclear on the purpose of this, as this is undefined due to being initialized to null in createImmediateNotificationEmailObject

Reply:
? It's set just a few lines up.